frontend/components/vault/VaultLiquidationTransactionFlow.vue

```diff
@@ -0,0 +1,147 @@
+                :is-liquidating="isLiquidating"
+                :is-explanations-shown="isExplanationsShown"
+                @liquidate="$emit('liquidate', $event)"
+            />
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { Alert, Button } from 'ant-design-vue';
+import { LiquidationLimits, VaultTransactionNotLiquidated } from 'auctions-core/dist/src/types';
+import TransactionFeesTable from '../auction/TransactionFeesTable.vue';
+import { generateLink } from '../../helpers/generateLink';
+import TimeTill from '../common/formatters/TimeTill.vue';
+import FormatAddress from '../common/formatters/FormatAddress.vue';
+import VaultLiquidationLimitsCheckPanel from '../panels/VaultLiquidationLimitsCheckPanel.vue';
+import VaultLiquidationPanel from '../panels/VaultLiquidationPanel.vue';
+import VaultLiquidationTransactionTable from './VaultLiquidationTransactionTable.vue';
+import WalletConnectionCheckPanel from '~/components/panels/WalletConnectionCheckPanel.vue';
+import TextBlock from '~/components/common/other/TextBlock.vue';
+import Explain from '~/components/common/other/Explain.vue';
+
+export default Vue.extend({
+    components: {
+        VaultLiquidationPanel,
+        VaultLiquidationLimitsCheckPanel,
+        FormatAddress,
+        TimeTill,
+        TransactionFeesTable,
+        VaultLiquidationTransactionTable,
+        TextBlock,
+        Alert,
+        Button,
+        WalletConnectionCheckPanel,
+        Explain,
+    },
+    props: {
+        vaultTransaction: {
+            type: Object as Vue.PropType<VaultTransactionNotLiquidated>,
+            required: true,
+        },
+        liquidationLimits: {
+            type: Object as Vue.PropType<LiquidationLimits>,
+            required: true,
+        },
+        walletAddress: {
+            type: String,
+            default: null,
+        },
+        isConnectingWallet: {
+            type: Boolean,
+            default: false,
+        },
+        isRefreshingLimits: {
+            type: Boolean,
+            default: false,
+        },
+        isLiquidating: {
+            type: Boolean,
+            default: false,
+        },
+        isExplanationsShown: {
+            type: Boolean,
+            default: true,
+        },
+    },
+    data() {
+        return {
+            isWalletConnected: false,
+            areLimitsNotReached: false,
+        };
+    },
+    computed: {
+        auctionLink(): string | undefined {
+            if (!this.vaultTransaction || !this.vaultTransaction.auctionId) {
+                return undefined;
+            }
+            const link = generateLink(this.vaultTransaction.network, 'collateral');
+            return `${link}&auction=${encodeURIComponent(this.vaultTransaction.auctionId)}`;
+        },
+        wasLiquidated(): boolean {
+            return this.vaultTransaction.state === 'liquidated';
+        },
```

Comment:
I am not sure why types allow this check, since `VaultTransactionNotLiquidated` doesn't have this kind of state. Types doesn't work?

Reply:
Types in my IDE never worked, but I am surprised it was not caught in the CI. However the comment above should fix it!

Reply:
> Types in my IDE never worked

OH WOW! Why was it never reported? Or did I miss it?